README.md

```diff
@@ -4,3 +4,52 @@ pol.is an AI powered sentiment gathering platform. More organic than surveys, le
 If you don't want to deploy your own instance of Polis, you can use our SaaS version [here](https://pol.is/home)
 Polis [can be easily embedded](http://docs.pol.is/usage/Embedding.html) on your page as an iframe.
 
+## Installation
+
+### Dependencies
+
+* PostgreSql `(>= 9.5.4.1)`
+* Node `6.11.1`
+We recommend installing [nvm](https://github.com/creationix/nvm) so you can easily switch between your favorite flavors of node.
+* NPM `3.3.8`
+
+### Setup
+
+1. Create a new datbase. You can name it whatever you please.
+    ```sh
+    create database polis;
```

Comment:
This didn't work for me, but `createdb polis` worked for me, does that work for you?

Reply:
With postgres 9.5, the following worked for me: 
   sudo -u postgres createdb polis